micro-rdk-installer/src/main.rs

```diff
@@ -190,7 +190,7 @@ fn request_wifi(
         ssid
     } else {
         Input::with_theme(&ColorfulTheme::default())
-            .with_prompt("Please enter WiFi SSID")
+            .with_prompt("Please enter WiFi SSID (WiFi Network Name) for the network that micro-rdk will connect to")
```

Comment:
If we are cleaning this up, maybe:
- "Please enter the"
-  put an `i.e.` inside the parens
- remove the capitals from `Network Name` since it isn't a proper noun
- Capitalize to `Micro-RDK` 

```
Please enter the WiFi SSID (i.e. WiFi network name) for the network that Micro-RDK will connect to
```